src/org/omegat/util/TMXWriter2.java

```diff
@@ -62,7 +62,7 @@
  * @author Aaron Madlon-Kay
  */
 public class TMXWriter2 implements AutoCloseable {
-    static String lineSeparator = System.lineSeparator();
```

Comment:
Could you extend `TMXWtier2` class to configure to use `system` line separator or `unix style` line separator with argument in constructor?  Please keep backward compatibility because plugins may use it.